generators/getMetric.js

```diff
@@ -0,0 +1,84 @@
+const path = require('path')
+const fs = require('fs')
+const { metricsList } = require('./getMetricFunctions')
+const fsExtra = require('fs-extra')
+
+const CURRENCY_DESCRIPTION = `
+* @param {string} currency The metric is calculated, using a currency of choice.
+* Available currencies: USD`
+
+const TIMEBOUND_DESCRIPTION = `
+* @param {string} timeBound The metric is calculated only by taking into account the
+* tokens/coins that have moved in the past number of years or days.`
+
+function generateGetMetric () {
+  let generatedDoc = ''
+  for (const metric of metricsList) {
+    generatedDoc += allTimeBoundTemplates(metric)
+  }
+  let file = path.join(`${__dirname}`, `../src/getMetric.js`)
+  if (fs.existsSync(file)) {
+    fsExtra.removeSync(file)
+  }
+  return fs.writeFileSync(file, generatedDoc)
+}
+
+function allTimeBoundTemplates (metric) {
+  metric.hasCurrency = metric.hasCurrency || false
+  metric.sheetMetricName = metric.sheetMetricName || metric.metric
+  metric.hasTimeBound = metric.hasTimeBound || false
+  let generatedString =
+    fillTemplate(metric.metric, metric.sheetMetricName, metric.description, metric.hasCurrency, metric.hasTimeBound)
+
+  return generatedString
+}
+
+function fillTemplate (metric, sheetMetricName, description, currency, timeBound) {
+  let [options, bonusDescription, functionArguments] =
+    optionsGenerator(currency, timeBound)
+  return `/**
+* Gets ${description}
+* @param {string} projectSlug Name of the asset at sanbase,
+* which can be found at the end of the URL (eg. the Santiment URL is
+* https://app.santiment.net/projects/santiment, so the projectSlug would be santiment).
+* @param {date} from The starting date to fetch the data. Example: DATE(2018, 9, 20)
+* @param {date} to The ending date to fetch the data. Example: DATE(2018, 9, 21)${bonusDescription}
```

Comment:
Who inserts the newline between the `to` param description and the `bonusDescription`? Is it commind from the definition of the `CURRENCY_DESCRIPTION` and the `TIMEBOUND_DESCRITPION`?

Reply:
Yes, it does, the multiline comments with `` does it.